specification/cognitiveservices/data-plane/ContentModerator/v1.0/ContentModerator.json

```diff
@@ -2865,114 +3400,397 @@
         }
       }
     },
-    "CreateJob": {
-        "properties": {
-          "jobId": {
-            "description": "Id of the created job.",
-            "type": "string"
-          }
+    "JobId": {
+      "properties": {
+        "jobId": {
+          "description": "Id of the created job.",
+          "type": "string"
+        }
+      }
+    },
+    "APIError": {
+      "type": "object",
+      "description": "Error information returned by the API",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Error"
+        }
+      }
+    },
+    "Error": {
+      "type": "object",
+      "description": "Error body.",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
         }
-      } 
+      }
+    }
   },
   "parameters": {
-    "languageQueryParameter": {
-      "name": "language",
+    "timescale": {
+      "name": "timescale",
+      "description": "Timescale of the video.",
+      "in": "query",
+      "required": false,
+      "type": "number",
```

Comment:
In this example this is an integer - if this is always an integer then it might be a good idea to avoid using "number" since that can cause precision errors when comparing two floats.

Reply:
Changed this type to integer

Reply:
Thanks @v-sodsou ! There are a couple other "timescale" definitions in the spec, should they also be defined as "integer"?